hub/management/commands/import_new_constituencies.py

```diff
@@ -71,9 +71,9 @@ def handle(self, quiet: bool = False, *args, **options):
         constituency_lookup = (
             get_dataset_df(
                 repo_name="2025-constituencies",
-                package_name="parliament_con_2025",
+                package_name="parliament_con_2024",
                 version_name="latest",
-                file_name="parl_constituencies_2025.csv",
+                file_name="parl_constituencies_2024.csv",
```

Comment:
Same for this, presumably these files have the same name.

Reply:
Fixed